Runtime/SimpleGraphQL/HttpUtils.cs

```diff
@@ -131,6 +131,7 @@ public static void Dispose()
                     if (webRequest.result == UnityWebRequest.Result.ConnectionError)
                     {
                         Debug.LogError("Error While Sending: " + webRequest.error);
+                        throw new UnityWebRequestException(webRequest);
```

Comment:
Would this make the console to crash?

Reply:
At line 177 there is the same thing (already there since the dawn of time).

If the flow I've found is the only one possible, then this exception is catched by our GraphQL utility, the good ol' `Query_Internal` method. Since it's a managed exception it shouldn't make the console crash.